src/Specular/Ref.purs

```diff
@@ -40,45 +50,43 @@ data Ref a = Ref (Dynamic a) ((a -> a) -> Effect Unit)
 instance invariantRef :: Invariant Ref where
   imap f g (Ref v update) = Ref (f <$> v) ((\h -> g <<< h <<< f) >>> update)
 
--- | Old name for `new`.
-newRef :: forall m a. MonadEffect m => a -> m (Ref a)
-newRef = new
 
 -- | Create a new Ref with an initial value.
 new :: forall m a. MonadEffect m => a -> m (Ref a)
 new initial = do
   {dynamic, modify} <- newDynamic initial
   pure $ Ref dynamic modify
 
-newRefWithEvent :: forall m a. MonadFRP m => a -> Event (a -> a) -> m (Ref a)
-newRefWithEvent initial extraUpdate = do
+newWithEvent :: forall m a. MonadFRP m => a -> Event (a -> a) -> m (Ref a)
+newWithEvent initial extraUpdate = do
```

Comment:
Hm, this can possibly be implemented using other primitives. Would be nice if we could delete it. (Not in this PR)